
Thu, 09 Dec 2010 18:18:57 GMT

MAKE:
mv fasta.python3-4.python3 fasta.python3-4.py
/usr/local/src/Python-3.1.2/bin/python3.1 -OO -c "from py_compile import compile; compile('fasta.python3-4.py')"
  File "fasta.python3-4.py", line 61
    print s[i:i+width]
          ^
SyntaxError: invalid syntax

0.28s to complete and log all make actions

COMMAND LINE:
/usr/local/src/Python-3.1.2/bin/python3.1 fasta.python3-4.pyo 250000

PROGRAM FAILED 


PROGRAM OUTPUT:

/usr/local/src/Python-3.1.2/bin/python3.1: can't open file 'fasta.python3-4.pyo': [Errno 2] No such file or directory
